frontend/src/modules/search/components/SearchBox.js

```diff
@@ -143,6 +144,30 @@ export class SearchBoxBase extends React.Component<InternalProps, State> {
         }
     }
 
+    static getDerivedStateFromProps(prevProps: InternalProps, state: Object) {
```

Comment:
Without the `getDerivedStateFromProps` method, the filter panel works as it should, but on the flip side, the status value in the search box does not get updated when you click on the status numbers on the progress menu.
Might you have a suggestion on how I can go about this? @adngdb